lib/meilisearch-rails.rb

```diff
@@ -157,28 +157,22 @@ def get_attributes(document)
         attributes.merge!(attributes_to_hash(@additional_attributes, document)) if @additional_attributes
 
         if @options[:sanitize]
-          sanitizer = begin
-            ::HTML::FullSanitizer.new
-          rescue NameError
-            # from rails 4.2
-            ::Rails::Html::FullSanitizer.new
```

Comment:
CI was failing here on Rails 7. I think this got changed again maybe. I've replaced this with `ActionView::Base.full_sanitizer` which is backwards compatible and stops meilisearch-rails from having to deal with this going forward. Should simplify things nicely.